src/App/Pages/Settings/SettingsPage/SettingsPageViewModel.cs

```diff
@@ -598,13 +602,37 @@ public void BuildList()
             if (_vaultTimeoutPolicy != null)
             {
                 var maximumTimeout = _policyService.GetPolicyInt(_vaultTimeoutPolicy, "minutes").GetValueOrDefault();
-                securityItems.Insert(0, new SettingsPageListItem
+                var timeoutAction = _policyService.GetPolicyString(_vaultTimeoutPolicy, "action");
+                if (maximumTimeout != default && timeoutAction != default)
                 {
-                    Name = string.Format(AppResources.VaultTimeoutPolicyInEffect,
+                    securityItems.Insert(0, new SettingsPageListItem
+                    {
+                        Name = string.Format(AppResources.VaultTimeoutPolicyWithActionInEffect,
+                        Math.Floor((float)maximumTimeout / 60),
+                        maximumTimeout % 60,
+                        timeoutAction == "lock" ? AppResources.Lock : AppResources.LogOut),
```

Comment:
IMO this `"lock"` value should be in the same constants file as the policies constants. And also IMO this should be tabbed or at first glance it might be confused with properties of the `SettingsPageListItem` instead of params of the `string.Format`

Reply:
👍